src/drive/web/modules/services/components/Picker/Picker.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from 'react'
+import { IntentHeader, Button, withBreakpoints } from 'cozy-ui/react'
+import { ROOT_DIR_ID } from 'drive/constants/config'
+import Topbar from 'drive/web/modules/Layout/Topbar'
+import FileList from 'drive/web/modules/filelist/FileList'
+import {
+  Breadcrumb,
+  PreviousButton
+} from 'drive/web/modules/navigation/Breadcrumb'
+import { connect } from 'react-redux'
+import withReduxStore from './withReduxStore'
+import AddFolderButton from './AddFolderButton'
+import OriginHint from './OriginHint'
+import {
+  openFolder,
+  getOpenedFolderId,
+  getVisibleFiles
+} from 'drive/web/modules/navigation/duck'
+import styles from './picker.styl'
+
+class Picker extends Component {
+  state = {
+    path: []
+  }
+
+  updateBreadcrumb = folder => {
```

Comment:
We're handling the breadcrumbs without a router here. I'm not super happy with the implentation, mostly because if we start the navigation somewhere else than in the root folder, it wouldn't really work. I think this is something we can refine when we work on the folder-moving.